src/coreclr/System.Private.CoreLib/src/System/MulticastDelegate.cs

```diff
@@ -14,29 +14,14 @@ namespace System
 {
     [ClassInterface(ClassInterfaceType.None)]
     [ComVisible(true)]
-    public abstract class MulticastDelegate : Delegate
+    public abstract partial class MulticastDelegate : Delegate
```

Comment:
One more nit: It would be nice to rename this file to MulticastDelegate.CoreCLR to follow the naming convention for split implementations. (Same for Mono.)